modules/pool/subgraph-mapper.test.ts

```diff
@@ -0,0 +1,91 @@
+import { subgraphToPrisma } from './subgraph-mapper';
+import { poolFactory, poolTokenFactory } from '../../test/factories';
+
+describe('subgraphToPrisma', () => {
+    const weightedPool = poolFactory.build({
+        poolType: 'Weighted',
+    });
+
+    const stablePool = poolFactory.build({
+        poolType: 'ComposableStable',
+        amp: '0.1',
+    });
+
+    const linearPool = poolFactory.build({
+        poolType: 'Linear',
+        wrappedIndex: 1,
+        upperTarget: '1',
+    });
+
+    const elementPool = poolFactory.build({
+        poolType: 'Element',
+        principalToken: '0x123',
+    });
+
+    const gyroPool = poolFactory.build({
+        poolType: 'GyroE',
+        alpha: '0.5',
+        tauAlphaX: '0.5',
+    });
+
+    const fxPool = poolFactory.build({
+        poolType: 'FX',
+        alpha: '0.5',
+    });
+
+    it('should return correct object for weighted pool', () => {
+        const result = subgraphToPrisma(weightedPool, 'MAINNET', 1, []);
+        expect(result.data.type).toBe('WEIGHTED');
+    });
+
+    it('should return correct object for stable pool', () => {
```

Comment:
add test for phantom stable, will probably fail

Reply:
added one and it was failing indeed.